### ISSUE TITLE
$.ajax calls to http://localhost:45678 fail but external requests work fine

### ISSUE THREAD

**Issue by theogray** [phonegap/phonegap-app-desktop#375]
Phonegap Desktop v0.1.7

Our app accesses our own back end services which during development are run from URLs like `http://localhost:45678/api/jobs`

This works fine when calls are made from an older version of Phonegap Developer App running on Android (v1.4.0), but with latest Phonegap Developer v1.5.3 on iOS these calls fail - $.ajax returns `{readyState:0, status:0, statusText:"error"}`

If I switch our app to use our live back end services (e.g. `https://example.com/api/jobs`) then calls work fine, but this means we can't test against our internal data on Phonegap Developer at the moment.

**Comment by hermwong** [comment]
Can you try replacing localhost with the IP address of your server.

**Comment by theogray** [comment]
@hermwong I have tried with IP address of PC to check, but the dev side is running on IIS Express so only binding to port numbers on `127.0.0.1` (and possibly `::1`) by the looks of it. I have also tried with a custom host (`mycomputer.example.com`) specified in the hosts file on a dev PC pointing at `127.0.0.1` (plus port number) and no joy there.

Moving to full IIS instead of IIS Express is not simple due to having to set that up on every dev PC and also only being able to bind one debugger at a time (so only able to debug one back end service at a time).

As this doesn't affect v1.4.0 of Phonegap Developer on Android (against the same Desktop App) I wonder whether this is an issue in latest versions of Phonegap Developer rather than a Phonegap Desktop issue. I would prefer not to upgrade Android app at the moment to see as I don't want to lose ability to test on both my devices.

Are you able to reproduce the problem or have an idea of what code change might have caused it?
